tools/VersionController/Models/VersionBumper.cs

```diff
@@ -160,7 +160,9 @@ private void UpdateRollupModuleManifest()
             var file = File.ReadAllLines(rollupModuleManifestPath);
             var pattern = @"ModuleName(\s*)=(\s*)(['\""])" + moduleName + @"(['\""])(\s*);(\s*)RequiredVersion(\s*)=(\s*)(['\""])" + _oldVersion + @"(['\""])";
             var updatedFile = file.Select(l => Regex.Replace(l, pattern, "ModuleName = '" + moduleName + "'; RequiredVersion = '" + _newVersion + "'"));
-            File.WriteAllLines(rollupModuleManifestPath, updatedFile);
+            var pattern2 = @"ModuleName(\s*)=(\s*)(['\""])" + moduleName + @"(['\""])(\s*);(\s*)ModuleVersion(\s*)=(\s*)(['\""])" + _oldVersion + @"(['\""])";
+            var updatedFile2 = updatedFile.Select(l => Regex.Replace(l, pattern2, "ModuleName = '" + moduleName + "'; ModuleVersion = '" + _newVersion + "'"));
+            File.WriteAllLines(rollupModuleManifestPath, updatedFile2);
```

Comment:
```C#
var patternReplaceRequiredVersion = @"ModuleName(\s*)=(\s*)(['\""])" + moduleName + @"(['\""])(\s*);(\s*)RequiredVersion(\s*)=(\s*)(['\""])" + _oldVersion + @"(['\""])";
var patternReplaceModuleVersion = @"ModuleName(\s*)=(\s*)(['\""])" + moduleName + @"(['\""])(\s*);(\s*)ModuleVersion(\s*)=(\s*)(['\""])" + _oldVersion + @"(['\""])";
var updatedFile = file.Select(l => Regex.Replace(l, patternReplaceRequiredVersion, $"ModuleName = '{moduleName}'; RequiredVersion = '{_newVersion}'"))
                      .Select(l => Regex.Replace(l, patternReplaceModuleVersion, $"ModuleName = '{moduleName}'; ModuleVersion = '{_newVersion}'"));
File.WriteAllLines(rollupModuleManifestPath, updatedFile);
```
Maybe better?

Reply:
Yes, indeed. Thanks for your suggestion.